src/stepDefinitions/sharingContext.js

```diff
@@ -200,7 +200,8 @@ const checkReceivedSharesExpirationDate = function(user, target, days) {
       httpHelper.checkOCSStatus(result, 'Could not get shares. Message: ' + result.ocs.meta.message)
       const shares = result.ocs.data
       const currentDate = new Date()
-      let expectedExpirationDate = new Date(currentDate.setDate(currentDate.getDate() + days))
+      const getDay = Number(currentDate.getDate()) + Number(days)
+      let expectedExpirationDate = new Date(currentDate.setDate(getDay))
```

Comment:
This change is necessary because the `new Date(currentDate.setDate(currentDate.getDate() + days))` is doing the string concatenation. so adding `7` days to `7` will give `77` instead  of `14` and tests were failing for this reason in the CI https://drone.owncloud.com/owncloud/web/21649/28/21